dashboard/app/controllers/api/v1/assessments_controller.rb

```diff
@@ -1,5 +1,7 @@
 class Api::V1::AssessmentsController < Api::V1::JsonApiController
   include LevelsHelper
+  load_and_authorize_resource :section
+  load_resource :script
```

Comment:
why not `load_and_authorize_resource` here as well?

Reply:
Everyone has access to a script so I didn't think it needed to be authorized? I can add it if it's best practice to always authorize.

Reply:
i believe so, but i'm relatively new to using CanCan. there's already an ability set up for scripts that you should be able to use: 

https://github.com/code-dot-org/code-dot-org/blob/de0038e674ac831a9f4d799a2b42ba8b93135ac8/dashboard/app/models/ability.rb#L193-L203